### ISSUE TITLE
Why was choosed the Github repository for this purpose?

### ISSUE THREAD

**Issue by mezoni** [dart-lang/dart_enhancement_proposals#14]
On the main page I read about that the this Github repository is similar to
[Python's PEPs][pep] process and [Scala's SIPs][sip].

I want to ask the questions:

- Why was choosed the Github repository for this purpose?
- Why for this purpose was not written web application using Dart platform?

---

As far as I know, the Python PEP uses its own software written in the Python language and don't propose to use the Github issues for tracking each proposal as does this project.  

Source code of this project is open and available here: https://github.com/python/peps

According to (PEP) documentation::
Each PEP must have a `champion` -- someone who writes the PEP using the style and format described below, `shepherds` the discussions in the `appropriate forums`, and attempts to `build community consensus` around the idea.

---

[Scala Improvement Process (SIP)][sip] also does not propose to use [Github issues][github] for this purpose.  
Scala Improvement Process uses its own web site for this purpose.

---

Dart platform is very useful software that was intended to write the web applications.
I was surprised when I learned that the Dart developer did not choose Dart platform for this purpose (to implement the tracker for proposals) but chose Github issues (issue per prpoposal).

[github]: https://github.com/
[pep]: https://www.python.org/dev/peps/
[sip]: http://docs.scala-lang.org/sips/

**Comment by munificent** [comment]
Asking why we did *not* do something, which you have done innumerable times over the years, is not productive. It adds no value to the Dart ecosystem.

**Comment by NDxTreme** [comment]
While I can't speak for the Dart team, Github is a respected and known quantity in the web space, that manages the complexity that such a piece of hand coded software would require. Yes, it could be done, but the goal is to get the discussion going, not to exhibit NIH (Not Invented Here) syndrome.

GitHub also has an association in the open source movement that the Dart team probably wanted to capitalize on.

TL:DR
Save time and complexity by using the write tool for the right job, and making it someone else's problem.

**Comment by mezoni** [comment]
>> Asking why we did not do something, which you have done innumerable times over the years, is not productive

At least, not very high quality of your work does not do you honor.
You never did anything for the community for free, but I always do it for free.
You always did everything for fee and only for your own benefit.
And, all the time, I trying to understand: Why even for money the Dart developers always did everything wrong all.

>> It adds no value to the Dart ecosystem.

Dart ecosystems already decay for long time.
I do not try help or hinder.
I want to know: Why you doing so few for the community?

**Comment by mezoni** [comment]
>> Asking why we did not do something, which you have done innumerable times over the years, is not productive.

This is productive because after the months or years after I ask, you made that in the same way as I propose.
But why you not made it as expected initially if you often redo your work?
You're honest organization and many people rely on your discretion.

**Comment by mezoni** [comment]
>> Asking why we did not do something, which you have done innumerable times over the years, is not productive. It adds no value to the Dart ecosystem.

I will always comment this your comment because you delete my comments on this your comment.
Or you can delete this your comment and I stop answer to you.
My comment:
This is productive, `asking why we did not do something` because Dart Team always do something initially wrong and later redo this work by many times.
This is more counterproductive than what I do.
I hope you'll ever understand it.

**Comment by mezoni** [comment]
You provoke me by the remove only my comment on your comment, but leaving your comment, without the possibility to make a clarification.
